task/common/machine/script.go

```diff
@@ -104,6 +104,17 @@ rclone copy "$RCLONE_REMOTE/data" /tmp/tpi-task
 
 yes | /etc/profile.d/install-driver-prompt.sh # for GCP GPU machines
 
+# FIX NVIDIA APT GPG KEYS (https://github.com/NVIDIA/cuda-repo-management/issues/1#issuecomment-1111490201) 🤬
+if test -f /etc/apt/sources.list.d/cuda.list; then
+  for list in cuda nvidia-ml; do mv /etc/apt/sources.list.d/$list.list{,.backup}; done
+  apt-get update
+  apt-get install --yes gpg
+  apt-key del 7fa2af80
+  apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/3bf863cc.pub
+  apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1404/x86_64/7fa2af80.pub
+  for list in cuda nvidia-ml; do mv /etc/apt/sources.list.d/$list.list{.backup,}; done
```

Comment:
btw since the commands just below have `sudo`, should these too?

```suggestion
  for list in cuda nvidia-ml; do sudo mv /etc/apt/sources.list.d/$list.list{,.backup}; done
  sudo apt-get update
  sudo apt-get install --yes gpg
  sudo apt-key del 7fa2af80
  sudo apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/3bf863cc.pub
  sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1404/x86_64/7fa2af80.pub
  for list in cuda nvidia-ml; do sudo mv /etc/apt/sources.list.d/$list.list{.backup,}; done
```

Reply:
None of them should: the whole script runs with `root` privileges